tests/arch/arm/arm_interrupt/README.txt

```diff
@@ -6,7 +6,9 @@ Description:
 
 The first test verifies that we can handle system fault conditions
 while running in handler mode (i.e. in an ISR). Only for ARM
-Cortex-M targets. The test also verifies
+Cortex-M targets.
+
+The test also verifies
```

Comment:
It would be nice to put a colon `:` at the end of this line, but that's not terribly important.